### PR TITLE
Gradle kts support

### PR DESCRIPTION
**Pull request by lszucs**

### CODE REVIEW

File: scanners/android/android.go [hunk 1]
```diff
@@ -40,7 +40,18 @@ func (scanner *Scanner) DetectPlatform(searchDir string) (_ bool, err error) {
 		return false, fmt.Errorf("failed to search for build.gradle files, error: %s", err)
 	}
 
-	return len(scanner.ProjectRoots) > 0, err
+	isRecognized := len(scanner.ProjectRoots) > 0
```
Comment (trapacska) [inline scanners/android/android.go]: I think this is not necessary, can be moved to the if simply, also then the return can be something like 
`return len(scanner.ProjectRoots) > 0, err`

Comment (lszucs) [reply scanners/android/android.go]: It is not immediately obvious, that `len(scanner.ProjectRoots) > 0` means, that the scanner successfully recognized a project. One must read and interpret the surrounding code. Using the properly named variable, this is not an issue.

File: scanners/android/android.go [hunk 2]
```diff
@@ -40,7 +40,18 @@ func (scanner *Scanner) DetectPlatform(searchDir string) (_ bool, err error) {
...
+
+	if !isRecognized {
```
Comment (trapacska) [inline scanners/android/android.go]: This way kotlin projects will be collected only if there was no android projects at all. It is up to the user if he mixes his project base languages. Please collect both android and kotlin based project paths.

Comment (trapacska) [reply scanners/android/android.go]: This way when the scanner asks the user which repos he want to select from, he will be able to select from both type.

File: scanners/flutter/flutter.go [hunk 3]
```diff
@@ -179,6 +179,14 @@ projects:
 			proj.hasAndroidProject = true
 		}
 
+		// todo: duplicate code; factor into detectAndroidProjectPath function?
```
Comment (lpusok) [inline scanners/flutter/flutter.go]: This is somewhat similar: https://github.com/bitrise-tools/go-android/blob/master/gradle/project.go

Comment (lszucs) [reply scanners/flutter/flutter.go]: Thanks, will discuss offline if it is usable here. TODO comment will be removed before merge.

**Pull request merged**